platforms/native/engine/jsb-gfx.js

```diff
@@ -56,7 +56,7 @@ deviceProto.copyTexImagesToTexture = function (texImages, texture, regions) {
 
 const oldDeviceCreateSwapchainFunc = deviceProto.createSwapchain;
 deviceProto.createSwapchain = function (info) {
-    info.windowHandle = window.windowHandler;
+    info.windowHandle = window.oh ? jsb.device.getWindowHandler() : window.windowHandler;
```

Comment:
The same as above.

Reply:
the same as they don't support eval on Native